src/or/onion.c

```diff
@@ -1339,3 +1342,466 @@ extended_cell_format(uint8_t *command_out, uint16_t *len_out,
   return 0;
 }
 
+/*****************************************************************************
```

Comment:
We don't generally use this header format in the rest of our codebase.  Then again, we don't necessarily have a consistent way to indicate sections of a module.